cilium/load_balancer/daemon/map.h

```diff
@@ -3,9 +3,12 @@
 
 #pragma once
 
+// Can't be within an extern "C" block.
+// Must be included prior to other BPF headers.
+#include "ebpf_api.h"

```

Comment:
Ok for this PR, but this indicates an issue in the main repository.  Can you file one?